edx_proctoring/api.py

```diff
@@ -868,18 +868,24 @@ def update_attempt_status(exam_id, user_id, to_status,
     in_completed_status = ProctoredExamStudentAttemptStatus.is_completed_status(from_status)
     to_incompleted_status = ProctoredExamStudentAttemptStatus.is_incomplete_status(to_status)
 
-    if in_completed_status and to_incompleted_status:
-        err_msg = (
-            u'A status transition from {from_status} to {to_status} was attempted '
-            u'on exam_id {exam_id} for user_id {user_id}. This is not '
-            u'allowed!'.format(
-                from_status=from_status,
-                to_status=to_status,
-                exam_id=exam_id,
-                user_id=user_id
-            )
+    illegal_status_transition_msg = (
```

Comment:
#NIT Can we please wrap these logic into a new function? I feel like the logic here is getting more and more complicated for validation.